.travis.yml

```diff
@@ -4,6 +4,13 @@ jdk:
   - oraclejdk8
 #  - openjdk9 # Not yet supported by travis
   - oraclejdk9
-script: mvn install -Dgpg.skip=true
+services:
+  - docker
+
+script:
+  - mvn install -Dgpg.skip=true
+  - pushd . && cd jml && docker build -t openjml . && popd
```

Comment:
Please run in the travis environment directly.

Also - does this need to run three times, on different JDKs? Ideally we'd prefer to just run this as a separate test step, instead of running it on each of the test matrix environments.

Reply:
> Ideally we'd prefer to just run this as a separate test step, instead of running it on each of the test matrix environments.

++ Both to only run this as often as is actually needed and to separate out "tests are failing" from "spec is failing/divergent", this should be run as a separate Travis environment.

Reply:
Yes, thanks for the feedback.  I'm not a Travis power-user, and am working on running the proof on a single JDK.  I'll try to figure out how to do it in a separate environment as well.

Reply:
Take a look at `matrix.include`. That lets you define exact combinations of configuration values for each separate environment.

https://docs.travis-ci.com/user/build-matrix/

example:

https://github.com/aws/aws-encryption-sdk-python/blob/master/.travis.yml

Reply:
Thanks.  Done.